<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MCNJY01" ignored="0" name="Test" tests="3" failures="2" timestamp="13 Jul 2019 18:00:01 GMT" time="23.967" errors="0">
  <testcase name="login" time="7.236" classname="datadriven.WordPress">
    <failure type="java.lang.AssertionError" message="expected [Log In ‹ opensourcecms — WordPress] but found [Dashboard ‹ opensourcecms — WordPress]">
      <![CDATA[java.lang.AssertionError: expected [Log In ‹ opensourcecms — WordPress] but found [Dashboard ‹ opensourcecms — WordPress]
	at datadriven.WordPress.login(WordPress.java:38)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="7.738" classname="datadriven.WordPress"/>
  <testcase name="login" time="5.922" classname="datadriven.WordPress">
    <failure type="java.lang.AssertionError" message="expected [Log In ‹ opensourcecms — WordPress] but found [Dashboard ‹ opensourcecms — WordPress]">
      <![CDATA[java.lang.AssertionError: expected [Log In ‹ opensourcecms — WordPress] but found [Dashboard ‹ opensourcecms — WordPress]
	at datadriven.WordPress.login(WordPress.java:38)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
